Sources/GRPCCore/Documentation.docc/Articles/Generating-stubs.md

```diff
@@ -83,3 +91,83 @@ swift build --product protoc-gen-grpc-swift
 
 This command will build the plugin into `.build/debug` directory. You can get the full path using
 `swift build --show-bin-path`.
+
+## Using the build plugin
+
+The build plugin (`GRPCProtobufGenerator`) is a great choice for convenient dynamic code generation, however it does come with some limitations.
+Because it generates the gRPC Swift stubs as part of the build it has the requirement that `protoc` must be guaranteed
+to be available at compile time. Also because of a limitation of Swift Package Manager build plugins, the plugin
+will only be invoked when applied to the source contained in a leaf package, so it is not useful for generating code for
+library authors.
+
+The build plugin will detect `.proto` files in the source tree and perform one invocation of `protoc` for each file 
+(caching results and performing the generation as necessary).
```

Comment:
Try to write in the active voice, e.g. "The build plugin detects `.proto` files in the source tree and invokes `protoc` once for each file."